mam/defs.h

```diff
@@ -24,6 +24,13 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#if defined(MAM_EXPORT_JS)
```

Comment:
use the preprocessor define.
```suggestion
#ifdef __EMSCRIPTEN__
```